docs/features/multiple-networks/multi-homing.md

```diff
@@ -1,33 +1,88 @@
 # Multihoming
-A K8s pod with more than one network interface is said to be multi-homed. The
-[Network Plumbing Working Group](https://github.com/k8snetworkplumbingwg/multi-net-spec)
-has put forward a [standard](https://github.com/k8snetworkplumbingwg/multi-net-spec)
-describing how to specify the configurations for additional network interfaces.
 
-There are several delegating plugins or meta-plugins (Multus, Genie)
-implementing this standard.
+## Introduction
 
-After a pod is scheduled on a particular Kubernetes node, kubelet will invoke
-the delegating plugin to prepare the pod for networking. This meta-plugin will
-then invoke the CNI responsible for setting up the pod's default cluster
-network, and afterwards it iterates the list of additional attachments on the
-pod, invoking the corresponding delegate CNI implementing the logic to attach
-the pod to that particular network.
+Multihoming allows the configuration of secondary-network interfaces for K8s pods.
+OVN-Kubernetes secondary-network has three configurable topologies: layer 3, layer 2, or localnet.
+
+- A layer 3 topology is a simplification of the topology for the cluster default network - but without egress.
+- A layer 2 topology supports east-west cluster traffic, but does not allow access to the underlying physical network.
+- A localnet topology allows connections to the physical network, but requires additional configuration of the
+  underlying Open vSwitch (OVS) bridge on cluster nodes.
+
+For layer 2 and localnet topologies, multihoming also allows IP features on secondary interfaces such as static IP
+allocation and persistent IP addresses for virtualization workloads.
 
-## Configuring secondary networks
 To allow pods to have multiple network interfaces, the user must provide the
 configurations specifying how to connect to these networks; these
-configurations are defined in a CRD named `NetworkAttachmentDefinition`.
+configurations are set in a CRD named `NetworkAttachmentDefinition`, defined by
+the [Kubernetes Network Custom Resource Definition De-facto Standard](https://github.com/k8snetworkplumbingwg/multi-net-spec).
+
+## Motivation
+
+Multihoming enables the workloads users (Pods and VMs) to send/receive traffic on network interfaces separate from the
+primary K8s network, without using complicated routing on the underlying physical network.
+It allows interconnecting the workloads through the OVN-Kubernetes logical switches to east-west workloads and to
+physical networks.
+
+For VM workloads, Multihoming can ensure that VMs retain their IP addresses even when they move across nodes, enhancing
+workload mobility and minimizing disruptions.
```

Comment:
Not sure I entirely understand.

>hm, I'm unsure this is the thing we want to highlight her

why not? `allowPersistentIPs` field is part of this feature, no? (really asking here :) )

>Maybe we should indicate why the kubernetes default network is usually not a fit for virt workloads ?

Can you elaborate why? I can think of several reasons (network performance, network isolation, needing complex/custom network topologies), so please focus me here.

Reply:
> Not sure I entirely understand.
> 
> > hm, I'm unsure this is the thing we want to highlight her
> 
> why not? `allowPersistentIPs` field is part of this feature, no? (really asking here :) )

Yes, but you're focusing on the tree, rather than in the forest. 

Multi-homing is useful if you need more than one interface on your pods. For whatever reason. You can then list useful / common use cases of multi-homing, including virt, where the cluster default network is **not** a fit. That's what I'm asking for below.

> 
> > Maybe we should indicate why the kubernetes default network is usually not a fit for virt workloads ?
> 
> Can you elaborate why? I can think of several reasons (network performance, network isolation, needing complex/custom network topologies), so please focus me here.

You usually can't live migrate, since the default cluster network will give your pod different IP addresses before / after migration.

> 
> > Still, this argument is good and should be kept. I'd probably just frame it differently, and indicate that ovn-k has/provides some virt-aware features for this type of workloads.
> 
> totally lost you there :) Can you be more specific?

I'm trying to say that after explaining why multi-homing is useful, and listing use cases for multi homing, in which you can go into some detail on some - namely virt - you could open the door to listing some virt specific features we have on ovn-k. Link to a different section where these are explained if you want.

But let's forget that for now and explain clearly why multi-homing is useful, and listing some use cases of that.

Reply:
rephrased. PTAL